src/main/java/com/notably/logic/commands/HelpCommand.java

```diff
@@ -11,9 +14,12 @@
     public static final String COMMAND_WORD = "help";
     public static final String COMMAND_SHORTHAND = "h";
 
+    private final Logger logger = LogsCenter.getLogger(getClass());
+
     @Override
     public void execute(Model notablyModel) {
         requireNonNull(notablyModel);
+        logger.info("Executing HelpCommand");
         notablyModel.setHelpOpen(true);
```

Comment:
Let's add another `logger.info()` after the `notablyModel.setHelpOpen(true)` execution? Just to be consistent with the other commands' logging

Reply:
Got it !